@here/harp-mapview-decoder/test/TileGeometryLoaderTest.ts

```diff
@@ -411,5 +396,26 @@ describe("TileGeometryLoader", function() {
                 expect(geometryLoader.isFinished).to.be.true;
             });
         });
+
+        // HARP-11468 Empty data should should still create ground plane if `isFullyCovering` true
+        it(`should create geometry for tile with covering DS and empty data`, async function() {
+            dataSource.addGroundPlane = true;
+            tile.decodedTile = undefined;
+            tile.tileLoader!.isFinished = true;
+
+            const geometryCreator = TileGeometryCreator.instance;
+            const spyAddGroundPlane = sandbox.spy(geometryCreator, "addGroundPlane") as any;
+            expect(spyAddGroundPlane.callCount).equal(0);
+
+            geometryLoader!.update();
+            expect(mapView.taskQueue.numItemsLeft(TileTaskGroups.CREATE)).equal(1);
+            expect(mapView.taskQueue.processNext(TileTaskGroups.CREATE)).equal(true);
+
+            await willEventually(() => {
+                expect(spyAddGroundPlane.callCount).equal(1);
+                // The geometry loader doesn't finish, because we expect the task to expire
```

Comment:
the comment is not matching the check in the next line .. "doesn#t finish.." ..isFinished).to.be.true

Reply:
Hahah, thanks.